docs/user-guide/install-overview.md

```diff
@@ -1,11 +1,14 @@
 # Overview
 
 The installation of Zowe&trade; consists of the following processes:
-- installation of the Zowe server components.
+
+- installation of the Zowe server-side components.
 
```

Comment:
Both "installation" bullets should start w/ a capital letter: "Installation of the Zowe ..."